zerver/openapi/zulip.yaml

```diff
@@ -2990,6 +2990,10 @@ paths:
                                       type: string
                                       description: |
                                         The description of the organization, used on login and registration pages.
+                                    realm_giphy_rating:
+                                      type: integer
+                                      description: |
+                                        Maximum rating of the GIFs that will be retrieved from GIPHY.
```

Comment:
I merged this first Giphy rating backend commit as 802c450b3f1b01fda63584cb1cab118de31c9f6c.  As part of doing so, I needed to add `**Changes**` entries in the OpenAPI docs to record the feature level where things were added.

Whenever you change anything in `zulip.yaml` that isn't just a bug fix, you need to document the changes so that clients like the mobile and terminal apps that are trying to support a range of versions can correctly implement support for the new feature and understand when it might be available.

Reply:
Okay, will keep that in mind.